app/src/ui/changes/confirm-commit-filtered-changes-dialog.tsx

```diff
@@ -42,8 +43,11 @@ export class ConfirmCommitFilteredChanges extends React.Component<
       >
         <DialogContent>
           <p id="confirm-commit-filtered-changes-message">
-            You have a filter applied. There are changes that will be committed
-            hidden from view. Are you sure you want to commit these changes?
+            You have a filter applied. There are{' '}
+            <LinkButton onClick={this.showFilesToBeCommitted}>
+              changes that will be committed hidden from view
```

Comment:
Maybe leave `hidden from view` out of the link? 🤔

Reply:
Maybe.. but I I wanted to link to communicate you can see the changes that are hidden

Maybe the sentence needs to change.   "There are `<link>`hidden changes `</link>` that will be committed." ?

Reply:
Love it! 💖